packages/asset-swapper/src/quote_consumers/exchange_proxy_swap_quote_consumer.ts

```diff
@@ -374,6 +375,56 @@ export class ExchangeProxySwapQuoteConsumer implements SwapQuoteConsumerBase {
             };
         }
 
+        //OTC orders
+
+        if(
+            [ChainId.Mainnet, ChainId.Polygon].includes(this.chainId) &&
+            !isToETH &&
+            !isFromETH &&
+            quote.orders.every(o => o.type === FillQuoteTransformerOrderType.Otc) &&
+            !requiresTransformERC20(optsWithDefaults)
+        ){
+            const otcOrdersData = quote.orders.map(o => o.fillData as NativeOtcOrderFillData);
+            const fillAmountPerOrder = (() => {
```

Comment:
nit/optional: maybe this could be refactored? 
looks the same as `fillAmountPerOrder` of the `RFQT VIP` logic above.